### PR TITLE
start to add initial instructions to readme.md file

### PR DESCRIPTION
**Pull request by h-chagas**
Not to sure about how the structure of the README.md should be and how to format it "prettier"

### CODE REVIEW

**Review by Augs0 — CHANGES_REQUESTED** [review]
Hey Hugo, you will need to instruct users what database names to put in their .env files and what to call them (e.g. .env.test), as well as reminding them to use .gitignore to prevent pushing them to GitHub. Finally, don't forget to install supertest and express :)

File: README.md [hunk 1]
```diff
@@ -1,57 +1,10 @@
-# Northcoders News API
+# Hugo's News API
 
-## Background
+## Initial instructions
+First of all, you must create two .env* files and populate them with the database names.
```
Comment (Poonam-raj) [inline README.md]: Would be good to say explicitly that they should create a file called '.env.development' and '.env.test' even though you reference them below we have to be super clear

File: README.md [hunk 2]
```diff
@@ -1,57 +1,10 @@
-# Northcoders News API
```
Comment (Poonam-raj) [inline README.md:1]: Please tell the user to run `npm install` too otherwise they'll encounter a load of bugs if they try to use the api

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
As well as the comments i've left on the readme, please make sure you've installed the dependencies we need. As well as what is already in the package.json, we have to have 'express' and 'supertest' - would they be dev or normal dependencies?
Also would 'pg-format' be needed in the dev side too? Might be worth moving pg-format into normal dependencies

**Review by Augs0 — APPROVED** [review]

**Pull request merged**